metadata-ingestion/src/datahub/ingestion/api/source_helpers.py

```diff
@@ -536,3 +541,29 @@ def _prepend_platform_instance(
         return [BrowsePathEntryClass(id=urn, urn=urn)] + entries
 
     return entries
+
+
+class TimeStampMetadata:
```

Comment:
The class does more than just stamping the time. How about renaming it to something more descriptive, like StampSystemMetadata or FallbackSystemMetadata?

Reply:
Renamed to AutoSystemMetadata